commandeer.nim

```diff
@@ -284,6 +284,26 @@ template subcommand*(identifier: untyped, subcommandName: string,
     statements
     currentMapping = subCommands[""]
 
+template subcommand*(identifier: untyped, subcommandNames: openArray[string],
+                    statements: untyped): untyped =
```

Comment:
Haha! I didn't set a good example above, but let's indent `statements` to match `identifier` (change the one above too if you like).